shared/src/vyxal/Elements.scala

```diff
@@ -3928,6 +3928,20 @@ object Elements:
     ) {
       case a: VNum => a / VNum(spire.math.Real.pi) * VNum(180)
     },
+    addPart(
+      Dyad,
+      "ÞR",
+      "Reshape",
+      List("reshape"),
+      false,
+      "a: lst, b: lst[num] => a reshaped to shape b",
```

Comment:
Just to be clear, this only works if `a` is a flat list, right? If I want to reshape a matrix, I'll have to flatten it first and then reshape it?

Reply:
that is correct

Reply:
Maybe something like `a (assumed flat) reshaped to shape b`? But then people might think you can't use it on nested lists at all